CHANGELOG.md

```diff
@@ -207,6 +214,11 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - **Mono:** Display stack traces for inner exceptions.
 - **Mono:** Bundle `mscorlib.dll` with Godot to improve portability.
 
+### Deprecated
+
+- [Renamed several core files](https://github.com/godotengine/godot/pull/25821).
```

Comment:
I guess this could go among "Changed" too, to avoid having it alone in its section. At least "Changed" already includes lots of compatibility changes.